test/test-mustache.js

```diff
@@ -376,6 +376,51 @@ test('reply.view with mustache engine with partials', t => {
   })
 })
 
+test('reply.view with mustache engine with partials in production mode should use cache', t => {
+  t.plan(11)
+  const fastify = Fastify()
+  const mustache = require('mustache')
+  const data = { text: 'text' }
+
+  fastify.register(require('../index'), {
+    engine: {
+      mustache: mustache
+    },
+    production: true
+  })
+
+  fastify.get('/', (req, reply) => {
+    reply.view('./templates/index.mustache', data, { partials: { body: './templates/body.mustache' } })
+  })
+
+  fastify.listen(0, err => {
```

Comment:
Here same thing I wanted somehow to "trigger" the page to be pulled from the cache but again no way on asserting on that or checking the cache that actually has items inside

Reply:
As previous comment, `proxyquire` can help you on this task :+1: